spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -35,10 +35,36 @@
   end
 
   describe '#update' do
+    let(:params) do
+      {
+        idv_how_to_verify_form:
+        { selection: selection },
```

Comment:
```suggestion
        idv_how_to_verify_form: { selection: selection },
```

It's been a long day and my brain is fried, but I think this should be one line, or else the value on line 41 should be indented? My brain parses this as a list otherwise.

Reply:
Done. See commit 1672de5a246c821175c5ed22bffb9ce73d281df6

Please resolve comment if happy with my change so I know you saw it